client/src/components/reviews/Reviews.jsx

```diff
@@ -156,7 +163,7 @@ class Reviews extends React.Component {
   render () {
     return (
         <div className='Reviews'>
-          {this.state.reviewList.length > 0 && (<Ratings filterRatings={this.filterRatings.bind(this)} ratingsBreakdown={this.state.ratingsBreakdown} starValue={this.props.starsValue} characteristics={this.state.characteristics}/>)}
+          {this.state.reviewList.length > 0 && (<Ratings filterRatings={this.filterRatings.bind(this)} ratingsBreakdown={this.state.ratingsBreakdown} removeFilterRatings={this.removeFilterRatings.bind(this)} filtered={this.state.filterRatings} starValue={this.props.starsValue} characteristics={this.state.characteristics} recommended={this.state.recommended}/>)}
```

Comment:
Can consider binding "this" in your constructor so there's less re-typing and improves readability. Also can consider deconstructing your objects at the top of your render() method (between line 163 and 164). Example const { filteredRatings, recommended } = this.state; Benefits being improved readability and less retyping :)